components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -490,7 +490,8 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
           return;
         }
       }
-    } else {
+    }
```

Comment:
So skip endpoint will not IngressType.Iap nor IngressType.BasicAuth?
Can we do else if (this.state.ingress === IngressType.BasicAuth) here, and then 
else {
// skip endpoint
log something
}

Reply:
Log here would be user facing as this part runs at user browser.
Do you mean print some messages for  "Setup Endpoint later" case?
There's a message field for each mode, we can use that to explain behavior of user's choice